### PR TITLE
Master dev virial

### PR DESCRIPTION
**Pull request by elambros**
added buckingham virial support

### CODE REVIEW

File: src/tests/single_point.cpp [hunk 1]
```diff
@@ -56,7 +56,7 @@ int main(int argc, char** argv)
     }
 
 
-    double en = systems[0].Energy(true);
+    double en = systems[0].Energy(true); // EL EL CHANGE THIS AFTER
```
Comment (chemphys) [inline src/tests/single_point.cpp]: What is this?

File: src/tests/unittests/CMakeLists.txt [hunk 2]
```diff
@@ -23,6 +23,7 @@ set( SOURCES_UNITTESTS_TESTS
     unittest-h2o-cl-dimer.cpp
     unittest-h2o-br-dimer.cpp
     unittest-h2o-virial.cpp
+    unittest-h2o-ion-virial.cpp
```
Comment (chemphys) [inline src/tests/unittests/CMakeLists.txt]: I think this is not ok... You have a unittest for na-h2o dimer. Maybe you could add the virial for that configuration in the header file and use that setup file. Also, there is a unittest na-h2o that should also test the virial in there. What I mean, merge your virial test into the na-h2o unittest.

File: src/tests/unittests/setup_na_h2o_virial.h [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    std::vector<double> virial_elec = { \
+        11.1457 , -3.97076 , 1.79961 , -3.97076 , 2.08396 , 1.0376 , 1.79961 , 1.0376 , 5.64599}; \
+    std::vector<double> virial_buck = {8.14904 , -3.9217 , -3.47649 , -3.9217 , 1.89835 , 1.69604 , -3.47649 , 1.69604 , 1.537}; \
+    std::vector<double> virial_disp = { \
+        -2.63753 , 1.24327 , 1.02596 , 1.24327 , -0.592478 , -0.501568 , 1.02596 , -0.501568 , -0.453768}; \
+    std::vector<double> real_coords = std::vector<double>{                                          \
+   0.0000029477,        0.0000014926,       -0.0000005021,\
+    -2.7779819073,        1.2138365139,        0.9901681838,\
+   -3.1030647065,        1.1366277718,        0.0870932044, \
+   -1.8474796079,        0.9364284622,        0.8766593013 };
```
Comment (chemphys) [inline src/tests/unittests/setup_na_h2o_virial.h]: These should be defined in the na-h2o dimer header file. Please recalculate the virial for the configuration defined there and use that as unittest!

File: src/tests/single_point.cpp (hunk 1 above)
Comment (elambros) [reply src/tests/single_point.cpp]: Oh, I had changed that for some testing and forgot to remove the comment. I'll remove it

**Comment by chemphys** [comment]
Seems good now. I will merge once travis passes,

**Pull request merged**